tests/foreman/api/test_ansible.py

```diff
@@ -301,9 +301,11 @@ def test_positive_read_facts_with_filter(
         host.organization = module_org
         host.location = module_location
         host.update(['organization', 'location'])
-        request.addfinalizer(
-            user.delete
-        )  # Adding a temporary workaround until the issue 'SAT-18656' is resolved.
+        if is_open('SAT-18656'):
```

Comment:
Could you leverage this PR to limit the parametrization for this test only for default rhel version?

Reply:
Sure